context/app/static/js/components/detailPage/StatusIcon/style.js

```diff
@@ -5,7 +5,7 @@ const ColoredStatusIcon = styled(LensIcon)`
   color: ${(props) => props.theme.palette[props.$iconColor].main};
   font-size: 16px;
   margin-right: 3px;
-  align-self: center;
+  margin-bottom: -2.5px;
```

Comment:
Can we create a component for typography with an icon which relies on flex to center the items instead of specific margin values?